clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -5767,6 +5797,10 @@ static void handleAssertCapabilityAttr(Sema &S, Decl *D, const ParsedAttr &AL) {
 
 static void handleAcquireCapabilityAttr(Sema &S, Decl *D,
                                         const ParsedAttr &AL) {
+  if (const auto *ParmDecl = dyn_cast<ParmVarDecl>(D))
+    if (!checkFunParamsAreScopedLockable(S, ParmDecl, AL))
```

Comment:
@AaronBallman's suggestion for `handleLocksExcludedAttr` might apply here as well (and below). But I'm fine either way.